### ISSUE TITLE
results_barcharts_species missing records

### ISSUE THREAD

**Issue by denislepage** [BirdStudiesCanada/NatureCounts-TO-DO#44]
Some collections (EBIRD, BBS and probably many others) had no records in results_barcharts_species and results_barcharts_shapes_species. 

I don't know if this is linked to the new proc [prCreateSummaryCharts], but I am testing it now. We'll have to keep an eye on this.

**Comment by denislepage** [comment]
There's an extensive list of collections missing records, as a matter of fact (407 of them):

select distinct collection from results_barcharts_totals a where not exists (select * from results_barcharts_species b  where a.collection = b.collection)

**Comment by cjardine-bsc** [comment]
Best I can tell there is nothing in the new summary proc that is causing problems. The most recent CMMN data sets I imported are all in the _species table. 

I just ran another one through prCreateSummaryCharts and all seems good. 

My best guess is that at some point the table was accidentally wiped and the collections that are in there now are ones that have recently been imported and run. 

I sent you a permission request so I can test and clean. 

On thing I noticed is that all the summary procs (old and new) us like instead of equals. 

collection like @collection

Which is probably okay.
